### ISSUE TITLE
Feature request: Organize request tabs into groups/folders

### ISSUE THREAD

**Issue by ArtVoro** [a85/POSTMan-Chrome-Extension#12231]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

As a postman user I want to organize open request tabs into groups/folders for better overview, especially if many tabs are open at same time.

### Describe the solution you'd like

If you have multiple request tabs open, drag and drop tabs on top of each other to create a group/folder for the session.

Example:
3 requests open from collection A, 3 requests open from collection B. Right now all 6 appear next to each other.
I want to drag and drop them to organize them in the tab bar for the session. Maybe dynamically create a 2nd line of tabs, where first line corresponds to the groups and 2nd line corresponds to the tabs within the group.

![Screenshot 2023-08-14 at 12 03 18](https://github.com/postmanlabs/postman-app-support/assets/39726533/a92928be-74fc-49b8-824c-93eec7b6a520)

**Comment by dlipford360** [comment]
yeah if we could group requests, just like I can group tabs on a web browser and leave alittle note on the grouped tab, I would absolutely love this.

**Comment by francodv** [comment]
➕ 1 to this. it would be really useful

**Comment by Adham-Abdelwahab** [comment]
+1. This would be very, very useful. Please

**Comment by davelondono** [comment]
+1 this would be fantastic to have

**Comment by metalrammstein** [comment]
+1, this feature would be highly appreciated

**Comment by jeromesalonkee** [comment]
+1

For now what I do is open multiple window as a workaround, but that's not ideal.

Another approach could be one tab layout per collection. A little bit like they just did with env variables.

**Comment by dlancy-kroger** [comment]
+1, great idea
